west.yml

```diff
@@ -26,7 +28,7 @@ manifest:
           - tinycrypt
           - picolibc
     - name: thingset-node-c
-      remote: thingset
-      revision: ac06ae5832d76582053ab82a650c9c25e80378de
+      remote: garethpotter
+      revision: f919d8cd4ed91b5278ce84c4a17edd560951fbb9
```

Comment:
This needs adjustments, as the relevant PR has been merge.d

FYI: You can use e.g. `pull/25/head` instead of the commit SHA to point west to a still open PR of the upstream repo.